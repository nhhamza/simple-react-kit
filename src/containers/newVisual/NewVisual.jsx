import React from 'react';
import ImageGallery from '../../components/imageGallery/ImageGallery';
import './NewVisual.scss';

const NewVisual = () => {
  return (
    <>
      <div className="new-virsual__image-galley">
        <ImageGallery />
      </div>
    </>
  );
};

export default NewVisual;
