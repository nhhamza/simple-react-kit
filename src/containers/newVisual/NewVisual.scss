.new-visual {
  width: 100%;
}

.new-virsual__image-galley {
  width: 50%;
}
