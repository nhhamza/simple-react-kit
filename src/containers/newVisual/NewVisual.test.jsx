import React from 'react';

import { shallow } from 'enzyme';

import NewVisual from './NewVisual.jsx';

describe('NewVisual component', () => {
  const wrapper = shallow(<NewVisual />);

  it('should render', () => {
    expect(wrapper.exists()).toMatchSnapshot();
  });
});
