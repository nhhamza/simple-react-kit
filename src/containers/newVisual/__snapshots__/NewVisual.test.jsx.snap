// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewVisual component should render 1`] = `true`;
